cmd/zoekt-indexserver/config.go

```diff
@@ -369,7 +367,7 @@ func executeMirror(ctx context.Context, cfg []ConfigEntry, repoDir string, paral
 			continue
 		}
 
-		stdout, stderr := loggedRun(ctx, cmd)
+		stdout, stderr := loggedRun(trace, cmd)
```

Comment:
🔨  apologies if I'm missing something, but if you're doing a `loggedRun`, which prints/loops through `cmd.Args`, why spend the effort in `createGithubArgsMirrorAndFetchArgs` above manually adding in parts of the args to the trace?